<launch>
  <node pkg="xqserial_server" type="xqserial_server" name="motor_driver">
    <param name="debug_flag" value="false"/>
    <param name="bar_flag" value="true"/>
    <param name="wheel_separation" value="0.679"/>
    <param name="wheel_radius" value="0.08255"/>
    <param name="max_speed" value="5.0"/>
    <param name="port_car" value="/dev/stm32Car"/>
    <param name="port_imu" value="/dev/stm32Imu"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="baselink_broadcaster" args="0 0 0.15 0 0 0 1 base_footprint base_link 50">
  </node>
</launch>
